CONTRIBUTING.md

```diff
@@ -233,38 +247,61 @@ In order to make the test classes more readable, here are a set of naming conven
+- Stubed objects (mocks with predefined behavior) must be named from their class (with an optional qualifier), and prefixed by `stub`.
+    E.g.: `stubDataProvider: DataProvider`, `stubReader: Reader`.
+- Mocked objects (mocks being verified) must be named from their class (with an optional qualifier), and prefixed by `mock`. 
+    E.g.: `mockListener: Listener`, `mockLogger: Logger`).
+- Fixtures (data classes or primitives with no behavior) must be named from their class (with an optional qualifier), and prefixed by `fake`.
+    E.g.: `fakeContext: Context`, `fakeApplicationId: UUID`, `fakeRequest: NetworkRequest`.
 - Other fields can be named on case by case basis, but a few rules can still apply:
     - If the field is annotated by a JUnit 5 extension (e.g.: `@TempDir`), then it should be named after the extension (e.g.: `tempOutputDir`).
     
-### Test Methods
+#### Test Methods
 
 Test methods must follow the Given-When-Then principle, that is they must all consist of three steps: 
 
 - Given (optional): sets up the instance under test to be in the correct state;
 - When (optional): performs an action — directly or indirectly — on the instance under test;
 - Then (mandatory): performs any number of assertions on the instance under test’s state, the mocks or output values. It must perform at least one assertion. 
 
-If present, these steps will always be intruded by one line comments, e.g.: `// Given`.
+If present, these steps will always be intruded by one line comments, i.e.: `// Given`, `// When`, `// Then`.
 
-Based on this principle, the test name should reflect the intent, and use the following pattern: `MUST expected behavior WHEN method() withContext`. To avoid being too verbose, `MUST` will be written `𝕄`, and `WHEN` will be written `𝕎`. The `withContext` part should be concise, and can have a trailing curly braces context section to avoid duplicate names (e.g.: `𝕄 create a span with info 𝕎 intercept() for failing request {5xx}`)
+Based on this principle, the test name should reflect the intent, and use the following pattern: `MUST expected behavior WHEN method() GIVEN context`. 
+To avoid being too verbose, `MUST` will be written `M`, and `WHEN` will be written `W`. The `context` part should be concise, and wrapped in curly braces to avoid duplicate names 
+(e.g.: `M create a span with info W intercept() {statusCode=5xx}`)
 
-Parameters shall have simple local names reflecting their intent, whether they use an `@Forgery` or `@Mock` annotation (or none).
+Parameters shall have simple local names reflecting their intent (see above), whether they use an `@Forgery` or `@Mock` annotation (or none).
 
 Here's a test method following those conventions: 
 
 ```kotlin
     @Test
-    fun `𝕄 forward boolean attribute to handler 𝕎 addAttribute()`(
-        @StringForgery(StringForgeryType.ALPHABETICAL) key : String,
-        @BoolForgery value : Boolean
+    fun `M forward boolean attribute to handler W addAttribute()`(
```

Comment:
we don't use unicode variants for `M` and `W` anymore?

Reply:
A lot of tests aren't using them (they're not simple enough to type), and they do raise a bunch of warning in AS that add noise to the editor. It's still readable without the variants, so let's make it simpler to use the convention

Reply:
They are also causing the following Kotlin compiler bug I've opened in the past https://youtrack.jetbrains.com/issue/KT-60748. Although it can be solved.